src/events/guildMemberAdd.ts

```diff
@@ -18,9 +18,10 @@ export default TypedEvent({
             .setColor("ORANGE")
             .setTitle("New Member")
             .setDescription(
-                `Welcome ${member.user.username} to the conaticus server\n` +
-                    "Use `/rolemenu` to choose your pings and languages roles\n" +
-                    "Enjoy your stay!"
+                `Welcome ${member.user.username} to Conaticus' discord server\n` +
+                    "Be sure to read our #rules and feel free to introduce yourself in #introductions." +
```

Comment:
Could you cut out this introductions channel? We can make it mentionable by adding a new special channel and adding some automation to lock in the user's introduction.

https://github.com/conaticus/boolean/blob/3f2f8f506801b4a2858265a3aa1b3412d9cc2080/src/database/channels.ts#L6-L27